python-flatten-list/flatten_numpy_flatten.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+matrix = [list(range(100))] * 4
+arr = np.array(matrix)
```

Comment:
This ends up being slightly wrong since we flatten `matrix` while the NumPy array is named `arr`.

NumPy has its own `np.arange()` so it feels a bit backwards to use `range()` here and then convert to a NumPy array later. I'd suggest either using the literal matrix from the article, or doing something like this instead:

```suggestion
matrix = np.arange(400).reshape(100, 4)
```